ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -2722,17 +2722,13 @@ tests:
   cron: 44 0 14 * *
   steps:
     cluster_profile: vsphere-connected-2
-    env:
-      FEATURE_SET: TechPreviewNoUpgrade
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-vsphere-ipi-static-ip
 - as: vsphere-ipi-static-ip-tp-f28-destructive
```

Comment:
```suggestion
- as: vsphere-ipi-static-ip-f28-destructive
```

Reply:
BTW, can you also add a z-stream upgrade job in openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.16.yaml